examples/chat/src/app.tsx

```diff
@@ -15,6 +24,15 @@ export function App() {
     group.addMember("everyone", "writer");
     const chat = Chat.create([], { owner: group });
     router.navigate("/#/chat/" + chat.id);
+
+    if (window.parent) {
+      waitForUpload(chat.id, me).then(() => {
+        window.parent.postMessage(
+          { type: "chat-load", id: "/chat/" + chat.id },
+          "*",
+        );
+      });
+    }
```

Comment:
@aeplay @gdorsi all this code for the homepage chat demo is gonna get confusing for those referencing the chat app as a guide. What can I do, make another chat app specifically for the homepage demo?

Reply:
What about extracting this logic away in a separate module to make it less visible?

Would that work?

Better to keep one example if possible.